### PR TITLE
docs: remove docs links

### PR DESCRIPTION
**Pull request by hifabienne**

### CODE REVIEW

**Comment by fforootd** [comment]
Can it be that the name is wrong from this PR 
image

File: proto/zitadel/system.proto [hunk 1]
```diff
@@ -303,10 +299,7 @@ service SystemService {
 
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
       tags: "failed events";
-      external_docs: {
-        url: "https://docs.zitadel.com/concepts#Software_Architecture";
-        description: "details of ZITADEL's event driven software concepts";
-      };
+
```
Comment (fforootd) [inline proto/zitadel/system.proto:302]:
```suggestion
```

**Review by fforootd — APPROVED** [review]

File: proto/zitadel/system.proto (hunk 1 above)
Comment (fforootd) [reply proto/zitadel/system.proto:302]: minor correction

**Comment by hifabienne** [comment]
> Can it be that the name is wrong from this PR, I changed it, feel free to revert. image

yes

**Review by fforootd — DISMISSED** [review]

**Review by livio-a — APPROVED** [review]

**Pull request merged**